llvm/test/CodeGen/NVPTX/compute-ptx-value-vts.ll

```diff
@@ -0,0 +1,45 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_20
+
+define <6 x half> @half6() {
```

Comment:
We may as well auto-generate the checks to make sure we're still generating sensible PTX for those oddly shaped vectors.